outport/process/executionOrder/transactionsExecutionOrder.go

```diff
@@ -275,9 +277,10 @@ func extractSCRsFromMap(txsHashes [][]byte, scrs map[string]*outport.SCRInfo) ([
 func extractRewardsFromMap(txsHashes [][]byte, rewards map[string]*outport.RewardInfo) ([]data.TxWithExecutionOrderHandler, error) {
 	result := make([]data.TxWithExecutionOrderHandler, 0, len(txsHashes))
 	for _, txHash := range txsHashes {
-		reward, found := rewards[string(txHash)]
+		txHashHex := hex.EncodeToString(txHash)
```

Comment:
why do we keep the hash in hex format? This degrades the performance of the node as the hex.EncodeToString is more CPU intensive than the regular cast to string.

Reply:
we need the transaction hash hex-encoded in order can send it over to the WS. 
If I will let it non hex I will have problems with the marshaller because the hash is used as a string key in map structures.

Reply:
agree, this does not degrade the performance of the validators as the code impacted is in `outport...` package